pkgs/development/python-modules/vunit/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, colorama
+, fetchPypi
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "vunit";
```

There seems to be a naming conflict with [vunit](https://github.com/gcavalcante8808/vunit) on pypi
```suggestion
  pname = "vunit-hdl";
```